README.md

```diff
@@ -67,6 +67,13 @@ This virtual environment can then be activated in new terminal sessions with:
 - `shakedown --help` is your friend.
 
 
+### Running in parallel
+
+Shakedown can be run against multiple DC/OS clusters in parallel either by setting the `DCOS_CONFIG_ENV` environmental variable to a unique file, eg:
```

Comment:
`either` suggests there are more than one ways to do this, but only one way is described.